### PR TITLE
Added Shanghai Chapter Openness activity info.

### PR DESCRIPTION
**Pull request by steven004**
- Updated Readme to add openness (Feb.16) activity page 
- Create Openness main page 
- Added materials and pictures related to the openness activity

### CODE REVIEW

File: Events/20190216-Openness.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+> Details, see: [Activity Post](http://www.huodongxing.com/event/4480463696800?td=5082797945342)
+
+
+## Agenda
+- ProtoSchool Openness 
+  - IPFS-Force Community Introduction - Huijuan Sun
+  - ProtoSchool & Shanghai Chapter Introduction - Steven Li
+  - LD Research Institute Introduction - Django Peng
+- Knowledge sharing
+  - Filecoin Installation and Running -  Penghui Yan
```
Comment (DjangoPeng) [inline Events/20190216-Openness.md:18]: @yph152 Please check the name of your talk

File: Events/20190216-Openness.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+> Sponser: IPFS-Force Community 
+> 
+> Details, see: [Activity Post](http://www.huodongxing.com/event/4480463696800?td=5082797945342)
+
+
+## Agenda
+- ProtoSchool Openness 
+  - IPFS-Force Community Introduction - Huijuan Sun
+  - ProtoSchool & Shanghai Chapter Introduction - Steven Li
+  - LD Research Institute Introduction - Django Peng
```
Comment (DjangoPeng) [inline Events/20190216-Openness.md]: `LD Research Introduction` is better.

File: Events/20190216-Openness.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- [Filecoin Study Team](20190216-Openness/FilecoinStudyTeam.pdf)
+
+
+## Pictures - have fun
+IPFS Force Community - the Sponser speaking - Huijuan Sun:
+![](20190216-Openness/IPFS-Force_Community.jpeg)
+<br>
+
+LD Research Institute Introduction - Django Peng
+![](20190216-Openness/TianjiGe.jpeg)
```
Comment (DjangoPeng) [inline Events/20190216-Openness.md]: We'd better use `LDResearch` instead of `TianjiGe` both for filename and docs.

**Review by DjangoPeng — CHANGES_REQUESTED** [review]
Thanks for your contribution. Most of them are pretty good, except some typo issues.

**Comment by yph152** [comment]
@DjangoPeng ok,that's no problem.

**Comment by DjangoPeng** [comment]
/LGTM

merged

**Pull request merged**